src/lib/ui/srv_status_component.rb

```diff
@@ -0,0 +1,168 @@
+require "yast"
+Yast.import "Service"
+Yast.import "UI"
+
+module UI
+  class SrvStatusComponent
+    include Yast::UIShortcuts
+    include Yast::I18n
+    include Yast::Logger
+
+    def initialize(service_name, reload: true, reload_callback: nil, enabled_callback: nil)
+      @service_name = service_name
+      @reload = reload
+      @reload_callback = reload_callback
```

Comment:
After using @jreidinger as rubber duck on irc. I decided to remove the reload_callback.